drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -39,6 +39,9 @@
 #if defined(CONFIG_SOC_SERIES_STM32WBX)
 #define RCC_CALC_MSI_RUN_FREQ(range) __LL_RCC_CALC_MSI_FREQ( \
 						range << RCC_CR_MSIRANGE_Pos)
+#elif defined(CONFIG_SOC_SERIES_STM32L0X) || defined(CONFIG_SOC_SERIES_STM32L1X)
+#define RCC_CALC_MSI_RUN_FREQ(range) __LL_RCC_CALC_MSI_FREQ( \
+						range << RCC_ICSCR_MSIRANGE_Pos)
 #else
 #define RCC_CALC_MSI_RUN_FREQ(range) __LL_RCC_CALC_MSI_FREQ( \
 			LL_RCC_MSIRANGESEL_RUN, range << RCC_CR_MSIRANGE_Pos)
```

Comment:
Can this be simplified using `LL_RCC_MSI_GetRange` ?
```
#define RCC_CALC_MSI_RUN_FREQ(range) __LL_RCC_CALC_MSI_FREQ(LL_RCC_MSI_GetRange())
```

Reply:
though, the difference in shift operation appears clearly from the RCC register bit name, as it is written in the line above and the line below

Reply:
I mean: we can replace these 3 alternate definitions by a single one:
```
#define RCC_CALC_MSI_RUN_FREQ(range) __LL_RCC_CALC_MSI_FREQ(LL_RCC_MSI_GetRange())
```

Reply:
ok,